src/qt/spinbutt.cpp

```diff
@@ -34,12 +35,32 @@ void wxQtSpinButton::valueChanged(int value)
     wxSpinButton *handler = GetHandler();
     if ( handler )
     {
-        wxCommandEvent event( wxEVT_SPIN, handler->GetId() );
+        wxSpinEvent event( wxEVT_SPIN, handler->GetId() );
         event.SetInt( value );
         EmitEvent( event );
     }
 }
 
+void wxQtSpinButton::stepBy(int steps)
+{
+    wxSpinButton* handler = GetHandler();
+    if ( !handler )
+        return;
+
+    int eventType = (steps < 0) ? wxEVT_SPIN_DOWN : wxEVT_SPIN_UP;
```

Comment:
Why have the redundant parentheses?
```suggestion
    int eventType = steps < 0 ? wxEVT_SPIN_DOWN : wxEVT_SPIN_UP;
```

Reply:
No functional reason, just felt at the time that it might make code easier to read. Removed now.